### PR TITLE
Adding cache expiry to the JS Widget

### PR DESCRIPTION
**Pull request by urfolomeus**
Setting `Cache-Control: max-age=` on the JavaScript widgets to ensure that they don't get stale.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,4 +23,4 @@ notifications:
 env:
   global:
   - secure: Gj0KHq07rpTOa/Gtt9Bzb8xcQvY4jfhctOTYD24YtnY3GggcjAVRC1Hm8nVJ5FyXm54hX8ZoCGu4ferDTrtJLvOHu3klet9vghQjWESnCgqZxP9TTaYagWf1hPv52vzd3A8AJd+7eJ6MXxCi6TmojNXsh5rbAjxl4zxfQcE0xok=
-  - SKIP_SLOW_TESTS=true
+  - SKIP_SLOW_TESTS=false
```
Comment (neoeno) [inline .travis.yml]: This intended?

**Comment by neoeno** [comment]
:+1:

File: .travis.yml (hunk 1 above)
Comment (urfolomeus) [inline .travis.yml]: Yes, we want to run all tests on Travis. I accidentally turned it off previously.

Comment (neoeno) [inline .travis.yml]: I did wonder when I first saw that it was turned off! Thought you were maybe just being charitable to Travis

**Comment by darokel** [comment]
:+1:

**Comment by urfolomeus** [comment]
Thanks @neoeno and @darokel :)

**Pull request merged**